easybuild/easyconfigs/a/ancestry/ancestry-1.0.0-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,35 @@
+# easybuild easyconfig
+#
+# John Dey <jfdey@fredhutch.org>
+#
+# Fred Hutchinson Cancer Research Center - Seattle Washington - US
+#
+easyblock = 'MakeCp'
+
+name = 'ancestry'
+version = '1.0.0'
+
+homepage = 'https://github.com/vibansal'
+description = """Fast individual ancestry inference from DNA sequence data leveraging allele frequencies
+ from multiple populations. iAdmix Using population allele frequencies for computing individual admixture
+ estimates"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+dependencies = [
+    ('binutils', '2.32'),
+    ('ncurses', '6.1'),
+]
```

Comment:
binutils as a dependency is a bit strange. I would say this module should really just go into `GCC` instead of `GCCcore`.

Reply:
building from the source was a bit more work than I had time for. The Makefile should be re-written and the repo has not been updated for years. Sorry, this is so lame, but people needed it.